deployment/gimbal-discoverer/02-kubernetes-discoverer.yaml

```diff
@@ -4,43 +4,43 @@ metadata:
   labels:
     app: kubernetes-discoverer
     cluster: k8s
-  name: k8s-kubernetes-discoverer
+  name: k8s-kubernetes-discoverer2
   namespace: gimbal-discovery
 spec:
   selector:
     matchLabels:
-      app: kubernetes-discoverer
+      app: kubernetes-discoverer2
       cluster: k8s
   replicas: 1
   template:
     metadata:
       labels:
-        app: kubernetes-discoverer
+        app: kubernetes-discoverer2
         cluster: k8s
       annotations:
         prometheus.io/scrape: "true"
         prometheus.io/port: "8080"
     spec:
       containers:
-      - image: gcr.io/heptio-images/gimbal-discoverer:master
+      - image: stevesloka/gimbal-discoverer:master
```

Comment:
Leftover change from testing?

Reply:
I'm too awesome at doing that. =) I just pushed a commit to revert them back.